### ISSUE TITLE
Importing different datasets with same prefix/suffix and different tags

### ISSUE THREAD

**Issue by mkghadban** [veusz/veusz#652]
Hi,
Thank you for the great plotting tool! I have been trying to import two different datasets using the same prefix or suffix but assigning a different tag to each dataset. Apparently, this is not working where the tag of dataset one is replaced by the tag of dataset two, and the columns in the dataset imported first are replaced by those imported from the dataset imported right after. For example, if have dataset 1 and dataset 2, and in each dataset, I have two columns x and y (same column names in both datasets), and I want to import dataset 1 with suffix "_my_suffix" under tag T1 and dataset 2 with same suffix "_my_suffix"  under tag T2, I should be getting the following in the dataset tree: 

- T1
   x_my_suffix     (from dataset 1)
   y_my_suffix     (from dataset 1)

- T2
   x_my_suffix    (from dataset 2)
   y_my_suffix    (from dataset 2)

But currently, I am getting

- T2  <replaced T1>
   x_my_suffix    (from dataset 2 <replaced dataset 1>)
   y_my_suffix    (from dataset 2 <replaced dataset 1>)

Is it possible to do that?

**Comment by korintje** [comment]
No. Since Veusz datasets are managed in a flat namespace, the datasets must have different names. Tags cannot be used to hierarchize datasets because multiple tags are intended to be assigned to a single dataset.

In order to import multiple datasets which have same header names, the prefix or suffix must be changed dynamically. `$FILENAME` is a special variable prepared for that purpose, allowing the original data file name to be included in the prefix or suffix. It can be used in conjunction with other characters by using like `_from__$FILENAME__`.

**Comment by clacknya** [comment]
Using $FILENAME could be a bit too long of a name. I hope there could be a more elegant solution.
